docs/src/pages/guides/styled-engine/styled-engine.md

```diff
@@ -33,6 +37,39 @@ By default, `@mui/material` has `@mui/styled-engine` as a dependency, but you ca
  }
 ```
 
+### npm
+
+As the package aliases do not work with npm at this moment, you need to update you bundler's config to add this alias. Here is an example of how you can do it, if you use `webpack`:
```

Comment:
It seems like this should work from npm v6.9? https://npm.community/t/release-npm-6-9-0/5911

Reply:
It doesn't work. See https://github.com/mui-org/material-ui/pull/27917, we needed to revert the changes for it.